### DISCUSSION TITLE
Brilliant

### DISCUSSION THREAD

**Discussion by nextlevel2023** [007revad/Synology_HDD_db#133] [General]
I just want to say this is absolutely brilliant, thank you

**Comment by 007revad** [comment]
Thanks.

**Comment by filcuk** [comment]
This has saved me... my heart is still racing from what I've just gone through.
Thanks a million.

**Reply by 007revad** [reply]
You're welcome.

**Comment by hoxlund** [comment]
bro!!!! I cannot thank you enough for this script.

I once again "lost" my 2 nvme storage pools on my ds920+. from a simple reboot.

Can't tell you how many hours I spent trying hundreds of linux commands trying to recovery my files.

In the end i gave up and ran your script expecting drives to come back but most likely not my files.

BAMMMM!

Drives came back, files good to go. And for good measure disabled the memory checking as I also tossed in a crucial 16gb module to give me 20gb capacity. Let's just say I started a backup routine on this day going forward just for good measure.

Everything works and now I have piece of mind that after future synology firmware updates and NAS reboots. This won't happen again.


Thank you so so much!!!!!

**Comment by jdevrie** [comment]
I have to agree. A brilliant script. I just created a second volume on my 2 m.2 Samsung SSD's which I couldn't do before because Synology only lets you create a storage pool on Synology SSD's. Keep up the good work.
